Mathlib/CategoryTheory/MorphismProperty/Presheaf.lean

```diff
@@ -0,0 +1,202 @@
+/-
+Copyright (c) 2024 Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne, Joël Riou, Ravi Vakil
+-/
+
+import Mathlib.CategoryTheory.Limits.Shapes.Pullback.CommSq
+
+/-!
+
+# Representable morphisms of presheaves
+
+In this file we define and develop basic results on the representability of morphisms of presheaves.
+
+## Main definitions
+
+* `Presheaf.representable` is a `MorphismProperty` expressing the fact that a morphism of presheaves
+  is representable.
```

Comment:
```suggestion
* `Presheaf.representable` is a `MorphismProperty` expressing the fact that a morphism `f` of presheaves
  is representable, i.e. that any morphism from a representable presheaf to the target can be pulled back
  along `f` to a morphism from a representable presheaf to the source.
```
Feel free to rephrase, I just think there should be a short description here of what this means mathematically.

Reply:
Good suggestion, I think I will spell it out a bit more even.